packit_service/worker/handlers/distgit.py

```diff
@@ -171,13 +171,10 @@ def sync_branch(
                     logger.info(f"Will retry for the {retries + 1}. time in {delay}s.")
                     # throw=False so that exception is not raised and task
                     # is not retried also automatically
+                    kargs = self.task.request.kwargs.copy()
```

Comment:
I think you ment `kwargs` not `kargs` here :D
otherwise this should work :pray:

Reply:
No, I did not use kwargs as the variable name for purpose but I want a name that recalls it (and it seems to me that it worked :-). I do not want to lead to potential future bugs using a standard keyword name for a different purpose.